src/config/preprocessing-config.yml

```
@@ -0,0 +1,14 @@
+#Set to True for cheap small data
+use_intraday: False
+from_days_ago: 24
+to_days_ago: 2
+#minimum number of transitions required to be included in functional network
+weight_threshold: 10
```

```suggestion
weight_threshold: 5
```
if it's gonna be the same as before https://github.com/alphagov/govuk-related-links-recommender/blob/dabe8a860c69d0fffc660c52ec4f89b6aa43a58f/src/data_preprocessing/query_content_id_edge_weights.sql#L66